specification/resources/gen-ai/definitions.yml

```
@@ -533,9 +578,14 @@ apiAgentTemplate:
       format: date-time
       type: string
     description:
-      description: Description of the agent template
+      description: Deprecated, moved to summary
```

```suggestion
      description: Description of the agent template. Deprecated, moved to summary
 deprecated: true
```
Think you can add the `deprecated` tag here as well.

---

What does "moved to summary" refer too?

---

Oh i see:
https://github.com/digitalocean/openapi/pull/1025/files/453845090dba8abd1cb17ec1be07efa91e149564#diff-185e430560815eeb55464b2b96c7d0e8f7cb4515ae989026ee795d4602860cc6R622-R625

Ya just make it more proper with a capital S